tests/conftest.py

```diff
@@ -97,3 +97,33 @@ def ssh(ssh_server):
         port=ssh_server.port,
         key_filename=key_path,
     )
+
+
+@pytest.fixture
+def temporary_windows_drive(repo_dir):
+    import string
+    from ctypes import windll
+    from win32con import DDD_REMOVE_DEFINITION
+
+    current_drive_name = os.getcwd()[0].upper()
+
+    new_drive_name = [
+        letter
+        for letter in string.ascii_uppercase
+        if letter != current_drive_name
+    ][0]
+    new_drive = "{}:".format(new_drive_name)
+
+    target_path = repo_dir.mkdtemp()
+
+    set_up_result = windll.kernel32.DefineDosDeviceW(0, new_drive, target_path)
```

Comment:
Btw, maybe we should try a few drives just in case you run into a drive that is already taken? E.g. say you are working on C and you for some reason have A, B taken, then your mount will fail.

Reply:
thats a good point